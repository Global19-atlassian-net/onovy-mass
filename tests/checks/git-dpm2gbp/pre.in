#!/bin/bash

# Git
git init >/dev/null
git add . >/dev/null
git rm -r --cached debian/ >/dev/null
git commit -m "import dummy_0.orig.tar.gz" >/dev/null
git checkout -b upstream >/dev/null 2>&1
pristine-tar commit ../dummy_0.orig.tar.gz >/dev/null 2>&1

# git-dpm
git-dpm init ../dummy_0.orig.tar.gz >/dev/null 2>&1

# debian/*
git add debian/* >/dev/null
git commit -m 'Debian dummy package' >/dev/null 2>&1
git-dpm prepare >/dev/null 2>&1
