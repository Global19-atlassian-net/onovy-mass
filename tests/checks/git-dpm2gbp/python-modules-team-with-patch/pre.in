#!/bin/bash

set -e

# Git
git init >/dev/null
git add . >/dev/null
git rm -r --cached debian/ >/dev/null
git commit -m "import dummy_0.orig.tar.gz" >/dev/null
git checkout -b upstream >/dev/null 2>&1
pristine-tar commit ../dummy_0.orig.tar.gz >/dev/null 2>&1

# git-dpm
git-dpm init ../dummy_0.orig.tar.gz >/dev/null 2>&1

# debian/*
git add debian/* >/dev/null
git commit -m 'Debian dummy package' >/dev/null
git-dpm prepare >/dev/null 2>&1

# patch
git-dpm checkout-patched >/dev/null 2>&1
echo "dummy2" >dummy
git add dummy
git commit -m 'dummy patch' >/dev/null
git-dpm update-patches >/dev/null 2>&1
