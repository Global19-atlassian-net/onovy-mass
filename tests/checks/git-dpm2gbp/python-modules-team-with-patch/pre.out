#!/bin/bash

# Not git-dpm
rm debian/.git-dpm

# Patches unapplied format
echo "dummy" >dummy
